customer-commons/src/main/java/no/unit/nva/customer/model/CustomerMapper.java

```diff
@@ -36,55 +31,13 @@ public static URI toId(UUID identifier) {
         return URI.create(NAMESPACE + "/" + identifier);
     }
 
-    /**
-     * Map from Customer from Db to Dto version.
-     *
-     * @param customerDb customerDb
-     * @return customerDto
-     */
-    public CustomerDto toCustomerDto(CustomerDb customerDb) {
-        CustomerDto customerDto = objectMapper.convertValue(customerDb, CustomerDto.class);
-        URI id = toId(customerDb.getIdentifier());
-        customerDto.setId(id);
-        customerDto.setContext(CONTEXT);
-        return customerDto;
-    }
-
-    /**
-     * Map from Customer from Db to Dto version without context object.
-     *
-     * @param customerDb customerDb
-     * @return customerDto
-     */
-    public CustomerDto toCustomerDtoWithoutContext(CustomerDb customerDb) {
-        CustomerDto customerDto = toCustomerDto(customerDb);
-        customerDto.setContext(NO_CONTEXT);
-        return customerDto;
-    }
-
-    /**
-     * Map from list of Customers from Db to Dto version.
-     *
-     * @param customersDbs list of CustomerDb
-     * @return customerList
-     */
-    public CustomerList toCustomerList(List<CustomerDb> customersDbs) {
-        List<CustomerDto> customerDtos = customersDbs.stream()
-            .map(this::toCustomerDtoWithoutContext)
-            .collect(Collectors.toList()
-            );
-        return CustomerList.of(customerDtos);
-    }
-
-    /**
-     * Map from Customer from Dto to Db version.
-     *
-     * @param customerDto customerDto
-     * @return customerDb
-     */
-    public CustomerDb toCustomerDb(CustomerDto customerDto) {
-        CustomerDb customer = objectMapper.convertValue(customerDto, CustomerDb.class);
-        return customer;
+    public static CustomerDto removeContext(CustomerDto customerDto) {
```

Comment:
Method for preserving the existing functionality. It will be removed once it is not used by CustomerList.